_includes/head.html

```diff
@@ -2,6 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/png" href="assets/imgs/favicon.png"/>
```

Comment:
Shouldn't this one also have `relative_url` in it? @rpannek

Reply:
Yes.

Reply:
I'll fix!